spec/rails_helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'rspec/rails'
 require 'capybara/rails'
 require 'factory_girl'
+require 'csv'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.